### ISSUE TITLE
Support synchronization with EteBase

### ISSUE THREAD

**Issue by urizev** [msoultanidis/quillnote#35]
It would be great to be able to this backend:

   https://www.etebase.com/

**Comment by msoultanidis** [comment]
Does it expose a REST API? If it does, is it one specifically for notes or just files?

**Comment by urizev** [comment]
Yes, it provides a documented REST API and also an open source SDK implementation for Android.

Initially, it was come up for contacts, calendar, task and notes but since version 2.0 it is a general purpose E2E synchronization protocol. As I said they provide the protocol, a backend server and client SDKs for some platforms.

It is very flexible since it fits for any data structure: notes, labels, tags, descriptions, friends, etc.

Check it out! I think it is really interesting.

I use it for my contacts and calendars but I haven't found a good note app supporting it.

**Comment by msoultanidis** [comment]
Okay, I'll take a look at it!
